bears/go/GoMetaLinterBear.py

```diff
@@ -0,0 +1,33 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gometalinter.v1',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'(.*):(\d*):(\d*):(.*) \((\w*)\)')
+class GoMetaLinterBear:
+    """
+    Checks the code using ``gometalinter``. This will run
+    some set of golang linters over each file separately.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {GoRequirement(
+        package='gopkg.in/alecthomas/gometalinter.v1', flag='-u')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting','Syntax','Missing import',
+                  'Unused Code', 'Smell', 'Unreachable Code','Security'}
+    SEE_MORE = 'https://github.com/alecthomas/gometalinter'
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
```

Comment:
go vet - Reports potential errors that otherwise compile.
go tool vet --shadow - Reports variables that may have been unintentionally shadowed.
gotype - Syntactic and semantic analysis similar to the Go compiler.
deadcode - Finds unused code.
gocyclo - Computes the cyclomatic complexity of functions.
golint - Google's (mostly stylistic) linter.
varcheck - Find unused global variables and constants.
structcheck - Find unused struct fields.
aligncheck - Warn about un-optimally aligned structures.
errcheck - Check that error return values are used.
megacheck - Run staticcheck, gosimple and unused, sharing work.
dupl - Reports potentially duplicated code.
ineffassign - Detect when assignments to existing variables are not used.
interfacer - Suggest narrower interfaces that can be used.
unconvert - Detect redundant type conversions.
goconst - Finds repeated strings that could be replaced by a constant.
gas - Inspects source code for security problems by scanning the Go AST.